kiwi/xml_state.py

```diff
@@ -1012,6 +1012,22 @@ def get_build_type_containerconfig_section(self) -> Any:
             return container_config_sections[0]
         return None
 
+    def get_installinitrd_modules(self, action: str) -> List[str]:
+        """
+        Gets the list of modules to append in installation initrds
+
+        :return: a list of dracut module names
+
+        :rtype: list
+        """
+        modules = []
+        installinitrd_sections = self.build_type.get_installinitrd()
+        for installinitrd_section in installinitrd_sections:
+            if installinitrd_section.get_action() == action:
+                for module in installinitrd_section.get_dracut():
+                    modules.append(module.get_module())
+        return modules
```

Comment:
Always returns a list. An empty one if no module for the given type (`add|omit|set`) is defined.

Reply:
got it